src/app/external-cluster-wizard/steps/external-cluster/provider/gke/component.ts

```diff
@@ -84,20 +115,58 @@ export class GKEClusterSettingsComponent
     this.control(Controls.Name).setValue(this._nameGenerator.generateName());
   }
 
+  isDialogView(): boolean {
+    return !this._nodeDataService.isInWizardMode();
+  }
+
+  onEnableAutoScalingChange(evt: MatCheckboxChange) {
+    if (!evt.checked) {
+      this.form.patchValue({
+        [Controls.MaxCount]: null,
+        [Controls.MinCount]: null,
+      });
+
+      this.control(Controls.MaxCount).updateValueAndValidity();
+      this.control(Controls.MinCount).updateValueAndValidity();
+    }
+  }
+
   private _initForm(): void {
+    const MIN_COUNT_DEFAULT_VALUE = 1;
+    const MAX_COUNT_DEFAULT_VALUE = 5;
     this.form = this._builder.group({
-      [Controls.Name]: this._builder.control('', [Validators.required, KUBERNETES_RESOURCE_NAME_PATTERN_VALIDATOR]),
+      [Controls.Name]: this._builder.control('', [Validators.required, GKE_POOL_NAME_VALIDATOR]),
```

Comment:
Do we need same `regex` in Cluster Creation one for GKE?

Reply:
yes 
![Screenshot from 2022-07-28 13-03-34](https://user-images.githubusercontent.com/85109141/181479683-70ae00f2-aec6-42eb-9676-551ee4aeca1b.png)